docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
 
   server:
     container_name: server
-    image: target/vela-server
+    image: localhost:5000/vela-server
```

Comment:
we should revert the changes made to the default docker-compose configurations

Reply:
Fixed.